lib/common/hash.go

```diff
@@ -109,7 +109,7 @@ func (h *Hash) UnmarshalJSON(data []byte) error {
 }
 
 // MarshalJSON converts hash to hex data
-func (h *Hash) MarshalJSON() ([]byte, error) {
+func (h Hash) MarshalJSON() ([]byte, error) {
```

Comment:
this is needed since there are telemetry message fields that uses `common.Hash` and fields that uses `*common.Hash` and when the message is marshaled the output to `common.Hash` is `[...]` and the output to `*common.Hash` is `0x...` and now the output for both uses is `0x...`